Makefile

```diff
@@ -22,7 +22,7 @@ check-style: | pre-run .npminstall
 clean:
 	@echo Cleaning app
 
-	npm cache clean
+	npm cache clean --force
```

Comment:
What are the makefile changes needed for?

Reply:
npm 5 complains, that's why I added it